SI/Sem.04/Pract.04/Task 03.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+bool isPositive(int num)
+{
+    if (num > 0)
```

Comment:
num > 0 is a boolean expression, so it returns a boolean value that you can directly return